ComplianceLevels.md

```diff
@@ -0,0 +1,20 @@
+# Compliance Levels
+
+The specificiation has various levels of compliance so that partner codes can
+implement various levels of specification based on their requirements and
+desired support.  
+
+## Tier 1
+- A) Verifies the JSON input
+- B) Can obtain energy/gradient/Hessian quantities
+- C) Can perform optization, response, frequency, and spectra computations and
+     record output 
+
+## Tier 2 
+- D) Records all miscellaneous quantities into the Variables field
+     (one-electron energy, two-electron energy, KS energy, ...) 
+- E) Exports Wavefunction quantities (orbitals, eigenvalues, densities, orbitals)
```

Comment:
too many orbitals :-)

Make clear that this is choose-your-own-Variables at this Tier, as opposed to the normalized version at Tier 3. This step is for intra-code-community programatic access.